packages/react-ui-ag/src/Modals/MoreOptionsModal.js

```diff
@@ -0,0 +1,170 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import themed from 'react-themed'
+import autobind from 'autobind-decorator'
+import classnames from 'classnames'
+import { Modal, Button, Text } from '@rentpath/react-ui-core'
+
+@themed(/^MoreOptionsModal/, { pure: true })
+
+export default class MoreOptionsModal extends PureComponent {
+  static propTypes = {
+    isOpen: PropTypes.bool,
+    theme: PropTypes.object,
+    moreOptionsForm: PropTypes.node,
+    total: PropTypes.number,
+    filterTotal: PropTypes.number,
+    closeModal: PropTypes.func,
+    onSubmit: PropTypes.func,
+    clearFilters: PropTypes.func,
+    restoreFilters: PropTypes.func,
+    className: PropTypes.string,
+  }
+
+  static defaultProps = {
+    theme: {},
+    clearFilters: () => {},
+    onSubmit: () => {},
+    closeModal: () => {},
+    restoreFilters: () => {},
+  }
+
+  getSubmitButton() {
+    const { filterTotal, total } = this.props
+
+    let count = (filterTotal !== null) ? filterTotal : total
+    let propertyCount = 'Show 0 Properties'
+
+    if (count > 1) {
+      count = this.formatNumberWithCommas(count)
+      propertyCount = `Show ${count} Properties`
+    } else if (count === 1) {
+      propertyCount = 'Show 1 Property'
+    }
+
+    return propertyCount
+  }
+
+  getPropertyCount() {
+    const { filterTotal, total } = this.props
+    let count = (filterTotal !== null) ? filterTotal : total
+    let propertyCount = '0 Properties Found'
+
+    if (count > 1) {
+      count = this.formatNumberWithCommas(count)
+      propertyCount = `${count} Properties Found`
+    } else if (count === 1) {
+      propertyCount = '1 Property Found'
+    }
+
+    return propertyCount
+  }
+
+  @autobind
+  handleSubmit() {
+    const {
+      onSubmit,
+      closeModal,
+    } = this.props
+
+    if (onSubmit) {
+      onSubmit()
+    }
+    closeModal()
+  }
+
+  @autobind
+  handleCancel() {
+    const {
+      closeModal,
+      restoreFilters,
+    } = this.props
+    restoreFilters()
+    closeModal()
+  }
+
+  formatNumberWithCommas(number) {
+    return String(number).replace(/(.)(?=(\d{3})+$)/g, '$1,')
+  }
+
+  renderHeader() {
+    const {
+      theme,
+    } = this.props
+
+    const propertyCount = this.getPropertyCount()
+
+    return (
+      <div className={theme.MoreOptionsModal_Header}>
+        <Text className={theme.MoreOptionsModal_PropertyCount} >
```

Comment:
We don't need this. `<Text/>` component already has a `.Text` class.

Reply:
It has been removed